spec/services/streams/twitch_access_token/get_spec.rb

```diff
@@ -24,7 +24,7 @@
 
     context "when there is an unexpired token in the cache" do
       it "returns the cached token" do
-        Rails.cache.write(described_class::ACCESS_TOKEN_AND_EXPIRATION_CACHE_KEY, ["FAKE_UNEXPIRED_TWITCH_TOKEN", 15.days.from_now])
+        RedisRailsCache.write(described_class::ACCESS_TOKEN_AND_EXPIRATION_CACHE_KEY, ["FAKE_UNEXPIRED_TWITCH_TOKEN", 15.days.from_now])
```

Comment:
These cache write methods are still writing the cached token the old way, as an array. We need to update them to use the new method to ensure the tests are working as expected. 

```ruby
RedisRailsCache.write(described_class::ACCESS_TOKEN_AND_EXPIRATION_CACHE_KEY, "FAKE_UNEXPIRED_TWITCH_TOKEN", :expires_in => 15.days.from_now)
```

Reply:
Thanks, missed it :)